mopidy/mpd/protocol/__init__.py

```diff
@@ -23,7 +23,7 @@
 LINE_TERMINATOR = '\n'
 
 #: The MPD protocol version is 0.17.0.
```

Comment:
Reverted this, as I am not sure if increasing it has any unintended side effects with some clients, and it was for testing why "changed: outputs" suddenly stopped working with the none mixer.